### PR TITLE
Functional tests - Add test "FO - Send message while logged out"

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | Functional tests - Add test "FO - Send message while logged out"
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| Related PRs       | N/A
| How to test?      | Execute `HEADLESS=false TEST_PATH="functional/FO/05*/03*" URL_FO="https://shop.tld/" npm run test:specific`

### CODE REVIEW

File: tests/UI/campaigns/functional/FO/05_contactUs/03_sendMessageAsAnonymous.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    await foLoginPage.goToFooterLink(page, 'Contact us');
+
+    const pageTitle = await foContactUsPage.getPageTitle(page);
+    await expect(pageTitle).to.equal(foContactUsPage.pageTitle);
+  });
+
+  it('should check if the email is empty', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'checkEmptyEmail', baseContext);
+
+    await foContactUsPage.selectByVisibleText(page, foContactUsPage.subjectSelect, contactUsData.subject);
```
Comment (nesrineabdmouleh) [inline tests/UI/campaigns/functional/FO/05_contactUs/03_sendMessageAsAnonymous.js]: You should create a method on 'contactUsPage' to select the subject and click on send button and not use them in the scenario.

File: tests/UI/campaigns/functional/FO/05_contactUs/03_sendMessageAsAnonymous.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    await page.click(foContactUsPage.sendButton);
+
+    const invalidEmailError = await foContactUsPage.getTextContent(page, foContactUsPage.alertDangerTextBlock);
+    await expect(invalidEmailError).to.contains(foContactUsPage.invalidEmail);
+  });
+
+  it('should check if the email is invalid', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'checkInvalidEmail', baseContext);
+
+    await foContactUsPage.selectByVisibleText(page, foContactUsPage.subjectSelect, contactUsData.subject);
```
Comment (nesrineabdmouleh) [inline tests/UI/campaigns/functional/FO/05_contactUs/03_sendMessageAsAnonymous.js]: Same here

File: tests/UI/campaigns/functional/FO/05_contactUs/03_sendMessageAsAnonymous.js (hunk 1 above)
Comment (Progi1984) [reply tests/UI/campaigns/functional/FO/05_contactUs/03_sendMessageAsAnonymous.js]: Fixed

File: tests/UI/campaigns/functional/FO/05_contactUs/03_sendMessageAsAnonymous.js (hunk 2 above)
Comment (Progi1984) [reply tests/UI/campaigns/functional/FO/05_contactUs/03_sendMessageAsAnonymous.js]: Fixed

**Comment by Progi1984** [comment]
@PrestaShop/qa-automation  It's rebased on `8.0.x` branch.

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thx @Progi1984 :)

**Pull request merged**